ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -77,7 +77,7 @@ Verify RHODS Installation
   IF  "${UPDATE_CHANNEL}" == "odh-nightlies" or "${cluster_type}" != "managed"
     IF  "${PRODUCT}" == "ODH"
         Apply DSCInitialization CustomResource    dsci_name=${DSCI_NAME}
-        Wait For DSCInitialization CustomResource To Be Ready    timeout=30
+        Wait For DSCInitialization CustomResource To Be Ready    timeout=200
```

Comment:
Quite a big bump - is this determined on an empirical evidence that it can take that long? I don't recall this particular thing should take that much time.

Reply:
In odh-e2e, when running the first test on fresh openshift local cluster, this can take quite a long time

```
2024-05-16T12:55:04 INFO  [Wait:61] Waiting for: DSCInitialization default-dsci readiness
2024-05-16T12:56:40 INFO  [LoggerUtils:61] Creating DataScienceCluster/test-codeflare
```

https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/gitops/job/rhoai-e2e/175/consoleFull

I suspect waiting for servicemesh in `istio-system` is what may take up this time

Reply:
We should improve the wait condition by waiting on pods in `redhat-ods-applications`

Reply:
Guys, could you, please, remove links to our internal infra?

Regarding the waiting condition - we can update it as Andrej proposes. But TBH, I tend to think it's not a bad thing to have this separated - with the current approach we can identify which exact step during installation went wrong.